### PR TITLE
MotionSensitivity attribute within the Eve Cluster

### PR DESCRIPTION
**Pull request by lboue**
`MotionSensitivity` attribute within the Eve Cluster

**AttributeId: 319422477** (0x00130a000d)
- 0 High
- 4 Medium
- 7 Low

image

### CODE REVIEW

**Comment by lboue** [comment]
Could you add a label or tell me how to do it?
`Error:  No labels defined on the pull request.`

**Comment by lboue** [comment]
Could you add a label?
Error:  No labels defined on the pull request.

File: matter_server/common/custom_clusters.py [hunk 1]
```diff
@@ -314,6 +318,31 @@ def attribute_type(cls) -> ClusterObjectFieldDescriptor:
 
             value: int = 0
 
+        @dataclass
+        class MotionSensitivity(
+            ClusterAttributeDescriptor, CustomClusterAttributeMixin
+        ):
+            """MotionSensitivity Attribute within the Eve Cluster."""
+
+            should_poll = True
```
Comment (marcelveldt) [inline matter_server/common/custom_clusters.py:327]: I'd rather not add more attributes to poll - is this really needed ?

**Review by marcelveldt — APPROVED** [review]
Thanks!

**Pull request merged**